### PR TITLE
Hindi demonstration video

### PR DESCRIPTION
**Pull request by shubhankarsharma876**
Added the tutorial video link in the readme file.

### CODE REVIEW

**Comment by sudiptob2** [comment]
Hey great effort! I saw the video.  There is a mistake though. You said it generates the stat in every 15 min, but by default it generates in every 24 hours. I will accept this PR not to block your contribution but, can you please update that ( like putting a text quote on the video regarding the mistakes) in the mean time ? 🙏

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
 # Installation
 
-## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md)
+## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 
+## For video tutorial demonstration (in Hindi) do [visit](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```
Comment (sudiptob2) [inline README.md:16]:
```suggestion
## Video Demonstration 
- [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@
 
 # Installation
 
-## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md)
+## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 
+## Video Demonstration
```
Comment (sudiptob2) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -12,7 +12,9 @@
 
 # Installation
 
-## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md)
+## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 
+## Video Demonstration 
+- [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```
Comment (sudiptob2) [inline README.md:17]:
```suggestion
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 # Installation
 
-## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md)
+## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md)
```
Comment (sudiptob2) [inline README.md:15]:
```suggestion
### For step-by-step screenshots go [here](docs/INSTALLATIONSTEPS.md) 
### Video Demonstration
  - [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]

**Comment by shubhankarsharma876** [comment]
Yes for sure no problem...

**Pull request merged**